airbyte-ci/connectors/metadata_service/lib/tests/fixtures/metadata_validate/invalid/metadata_registry_no_type_override.yaml

```diff
@@ -5,7 +5,7 @@ data:
   connectorType: source
   dockerRepository: airbyte/image-exists-1
   githubIssueLabel: source-alloydb-strict-encrypt
-  dockerImageTag: 0.0.1-exists
+  dockerImageTag: 0.0.1
```

Comment:
more cleanup from the previous changes to tests - this time for the `metadata_validate` files